docs/server/configuration.md

```diff
@@ -0,0 +1,68 @@
+
+Most likeley, you won't need to change this setting, unless you're sharing the machine with others (e.g. on a shared (v)server) or run several servers on the machine with different addresses.
+
+### port
+`port` (active by default, default: `8000`, accepting numbers from 1 to 2^16 -1) tells the server on which port to listen.
+The default is set to `8000`, a [registered port commonly used for the Django Development Webserver](https://en.wikipedia.org/wiki/List_of_TCP_and_UDP_port_numbers#Registered_ports).
+You can freely adjust this setting according to your server's setup and will need to, if another application already listens on `8000`).
+In case you need to change the port, you might consider to choose a private port (>`49151`).
+
+Be sure that the port you use is not used by any other application and is also accessible from outside of the machine.
+
+### socket
+`socket` (inactive by default, default: `/tmp/planarally.sock`, accepting filepath) tells the server where to look for an internal endpoint for communications with the network/internet.
+This is only needed if you want to establish communication in between PlanarAlly and the users via websockets.
+
+### Secure Sockets Layer (SSL)
+PlanarAlly supports to encrypt communication in between the server and the players via [Secure Sockets Layer (SSL)](https://en.wikipedia.org/wiki/Transport_Layer_Security).
+Despite the name, this is independent from the *socket* configuration, above.
+The SSL-configuration needs three variables to be configured: `ssl`, `ssl_fullchain`, and `ssl_privkey`.
+
+`ssl` (active by default, default: `false`, accepting true/false) tells the server whether SSL should be used.
+If set to `false` (default), the other two settings will simply be ignored as they are not needed by the server.
+**Despite it is set to false by default, it is highly recommended to use transport encryption.**
+
+`ssl_fullchain` (active by default, default: `cert/fullchain.pem`, accepting filepath) tells the server where to look for the `*.pem`-file containing the certificate containing the 'full chain' of certificates to your certificate authority (CA).
+Please note that neither the folder `cert`, nor the file `fullchain.pem` are shipped with PlanarAlly, but have to be generated/referred to for each individual installation by yourself.
+
+`ssl_privkey`(active by default, default: `cert/privkey.pem`, accepting filepath) tells the server where to look for the `*.pem`-file containing your private certification key.
+Please note that neither the folder `cert`, nor the file `privkey.pem` are shipped with PlanarAlly, but have to be generated/referred to for each individual installation by yourself.
+**Please also take the 'private' in 'private key' *very* seriously!**
+
+### Cross Origin Resource Sharing (CORS)
+PlanarAlly supports Cross Origin Resource Sharing.
+Yet, this functionality is deactivated by default.
```

Comment:
The 'yet' makes it seem as if this is a less-ideal choice to me.

When not specified only the same origin is allowed, which is a security measure,